assets/queries/openAPI/property_defining_maximum_not_greater_than_minimum/query.rego

```diff
@@ -0,0 +1,60 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	[path, value] := walk(doc)
+	openapi_lib.is_numeric_type(value.type)
+	minimum := value.minimum
+	maximum := value.maximum
+	minimum > maximum
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("%s.type", [openapi_lib.concat_path(path)]),
```

Comment:
I think we can change here to:
```suggestion
		"searchKey": sprintf("%s", [openapi_lib.concat_path(path)]),
```
Since the problem is not on type, but on the fact the definition of max and min is wrong